### ISSUE TITLE
Deploying a service based on Turi Create

### ISSUE THREAD

**Issue by jmizgajski** [apple/turicreate#1105]
Hey guys,

I am an avid user from the pre-Apple aquistion era. I have been meaning to ask if you plan to bring back a "server centric" deployment option - for example for a recommender system it really makes little sense to deploy it via CoreML to an IPhone would rather have a recommendation server that can be queried. That was available in the paid Turi offering prior to Apple. Are you planning to make it avaiable in the opensource Turi Create?

**Comment by TobyRoseman** [comment]
Hi @jmizgajski - Thanks for using Turi Create. Please be a little more specific about your use case. Do you basically want an easy way to turn a Turi Create Model into a web service? Or are you trying to do something different?

**Comment by jmizgajski** [comment]
yup turning a recommendation model into a web service. This used to be done with Turi/Graphlab Server and code was deployed to Graphlab Server via an annotation `@` in a notebook. also this server then handled re-training and stats for the model.

**Comment by jmizgajski** [comment]
I know I can just write a Flask WS in front of it, but this used to be available and now is missing.

**Comment by johnyquest7** [comment]
I believe that you can deploy it as a Flask service. My github has an example for Flask deployment.
